### ISSUE TITLE
Schedule post to publish?

### ISSUE THREAD

**Issue by StevensLabs** [lauraquellmalz/Scheduled-Content-Actions#15]
Why is publishing not an option? I would like to be able to draft then publish on a timed schedule, how do we incorporate this?

**Comment by fvdm** [comment]
Publishing on a schedule is a built in feature of WordPress, in the post's Status & visibility box.

**Comment by StevensLabs** [comment]
Correct, but I cannot schedule with this plugin a series of publish/drafts. I would have to come back after it drafted to re-schedule.

**Comment by andrasmalmos** [comment]
same problem.I need publish option too.
